### ISSUE TITLE
Contradiction in Language Spec w.r.t. XML Processing Instructions

### ISSUE THREAD

**Issue by scabug** [scala/bug#5109]
The Scala Language Specification v. 2.9, in section 10.1, 3rd paragraph (towards the end of page 131) reads:

"[...] Scala does not support [...], CDATA sections or processing instructions. [...]"

But the syntax section following this paragraph contains a production XMLContent using non-terminals "CDSect" and "PI", which refer to CDATA sections and processing instructions respectively.

Indeed, both CDATA and Processing Instructions are supported by the 2.9 versions of the language:

```scala
Welcome to Scala version 2.9.0.1 (Java HotSpot(TM) Server VM, Java 1.6.0_24).
Type in expressions to have them evaluated.
Type :help for more information.

scala> <?hi world?><t><![CDATA[text]]></t>
res0: scala.xml.NodeBuffer = ArrayBuffer(<?hi world?>, <t>text</t>)
```

**Comment by ashawley** [comment]
Agree, CDATA and PI are accepted by the compiler:

```
scala> <![CDATA[text]]>
res0: scala.xml.PCData = <![CDATA[text]]>

scala> <?x y?>
res1: scala.xml.ProcInstr = <?x y?>
```

Declarations are not, though:

```
scala> <!DOCTYPE html>
<console>:1: error: in XML literal: '-' expected instead of 'D'
       <!DOCTYPE html>
         ^
```

**Comment by som-snytt** [comment]
My comment from 2015 was from when I fixed the related issue #3368 for CDATA support; I think it was supposed to remind me to update the spec, but I forgot. Or possibly, I felt I couldn't top "child support."

**Comment by ashawley** [comment]
I read your comment twice, earlier, but didn't catch the humor until now.  The joke holds up, though.
